Sun Nov  4 18:28:29 FET 2012  Sergei Trofimovich <slyfox@community.haskell.org>
  * allow split-0.2
diff -rN -u old-diagrams-cairo/diagrams-cairo.cabal new-diagrams-cairo/diagrams-cairo.cabal
--- old-diagrams-cairo/diagrams-cairo.cabal	2012-11-04 18:29:06.849369361 +0300
+++ new-diagrams-cairo/diagrams-cairo.cabal	2012-11-04 18:29:06.851369361 +0300
@@ -37,7 +37,7 @@
                        cairo >= 0.10.1 && < 0.13,
                        cmdargs >= 0.6 && < 0.10,
                        gtk >= 0.12.0  && < 0.13.0,
-                       split >= 0.1.2 && < 0.2
+                       split >= 0.1.2 && < 0.3
   if !os(windows)
     cpp-options: -DCMDLINELOOP
     Build-depends:     unix >= 2.4 && < 2.6
